src/vs/workbench/services/keybinding/electron-browser/keybindingService.ts

```diff
@@ -200,7 +201,7 @@ function isValidContributedKeyBinding(keyBinding: ContributedKeyBinding, rejects
 
 let keybindingType: IJSONSchema = {
 	type: 'object',
-	default: { command: '', key: '' },
+	default: { command: '', key: '', args: [''] },
```

Comment:
not sure about the default value for args... i'd this is a special use-case and maybe doesn't deserve a default